src/display_list.rs

```diff
@@ -135,6 +135,10 @@ impl Drop for DisplayList {
     }
 }
 
+// `DisplayList`s may be used by multiple threads simultaneously
+unsafe impl Send for DisplayList {}
+unsafe impl Sync for DisplayList {}
```

Comment:
It would be better to provide a multi-threaded `DisplayList` test case.

Reply:
https://github.com/messense/mupdf-rs/commit/07c13b68adae47413c8efa2fb8f7f1b8121c8fb8